calico_test/tests/st/utils/docker_host.py

```diff
@@ -178,13 +189,20 @@ def start_calico_node_with_docker(self):
             etcd_auth = "%s:2379" % get_ip()
             ssl_args = ""
 
+        # If the hostname has been overridden on this host, then pass it in
+        # as an environment variable.
+        if self._hostname:
+            hostname_args = "-e HOSTNAME=%s" % self._hostname
+        else:
+            hostname_args = ""
+
         self.execute("docker run -d --net=host --privileged "
                      "--name=calico-node "
-                     "-e IP=%s -e ETCD_AUTHORITY=%s "
+                     "%s -e IP=%s -e ETCD_AUTHORITY=%s "
```

Comment:
oh. kinda wierd and annoying that we do 2 vars on the same line and the rest we don't. I guess you're just following suit. Wanna split all three of these for clarity?